src/com/esotericsoftware/kryo/util/IdentityObjectIntMap.java

```diff
@@ -523,6 +523,15 @@ public void ensureCapacity (int additionalCapacity) {
 		if (sizeNeeded >= threshold) resize(ObjectMap.nextPowerOfTwo((int)(sizeNeeded / loadFactor)));
 	}
 
+	/**
+	 * This method returns the current size of the map.
+	 *
+	 * @return The size of the map.

```

Comment:
Don't repeat yourself.

Reply:
But I love hearing the sound of my own voice! I'll fix it.